packages/node_modules/nav-frontend-lastestriper/src/lastestriper.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import * as PT from 'prop-types';
+import 'nav-frontend-lastestriper-style';
+
+export interface NavFrontendLastestriperProps {
+    /**
+     * Antall lastestriper. Default er 3.
+     */
+    linjer?: number;
+    /**
+     * Inline stiling
+     */
+    style?: object;
+    /**
+     * Egendefinert klassenavn.
+     */
+    className?: string;
+}
+
+class NavFrontendLastestriper extends React.Component<NavFrontendLastestriperProps, {}> {
+
+    lastestriper (linjer: number): React.ReactNode[] {
+        return Array(linjer)
+            .fill(0)
+            .map((_, indeks) => {
+                const idx = indeks % 3;
+                if (idx === 0) {
+                    return <div className="lastestripe" key={indeks} />;
+                    // tslint:disable-next-line:no-else-after-return
+                } else if (idx === 1) {
+                    return <div className="lastestripe lastestripe__kort_forsinkelse" key={indeks} />;
+                } else {
+                    return <div className="lastestripe lastestripe__lang_forsinkelse lastestripe__smal" key={indeks} />;
+                }
+            });
+    };
+
+    render() {
+        const {linjer, style} = this.props;
+        return (
+            <div className="nav-frontend-lastestriper" style={style}>
+                {this.lastestriper(linjer ? linjer : 3)}
```

Comment:
Om man fjerner optional fra `this.props.linjer`, og legger til `defaultProps` så slipper man denne sjekken

Reply:
Godt forslag! En defaultProps er lagt inn.